package.json

```diff
@@ -24,7 +24,7 @@
     "dependencies": {
         "@ethersproject/experimental": "5.4.0",
         "@ethersproject/providers": "5.4.5",
-        "@hai-on-op/sdk": "1.2.0",
+        "@hai-on-op/sdk": "1.2.0-99dc1efc.0",
```

Comment:
let's merge the sdk before merging this PR

Reply:
[not yet](https://www.npmjs.com/package/@hai-on-op/sdk?activeTab=versions), we need to merge this pr first  https://github.com/hai-on-op/sdk/pull/6